docs/README.md

```diff
@@ -37,6 +37,52 @@ See [Protocol Versions](01-Protocol-Versions) for specifics.
 
 See [the PHP library documentation](https://github.com/paragonie/past/blob/master/docs/02-PHP-Library).
 
+## What are PAST's design goals?
+
+### 1. Resistance to Implementation Error / Misuse
+
+While it will be possible for motivated developers to discover novel ways to
+make any tool insecure, PAST attempts to make it easier to develop secure
+implementations than to develop insecure implementations of the standard.
+
+To accomplish this goal, we cast aside runtime protocol negotiation and
+so-called "algorithm agility" in favor of pre-negotiated protocols with
+version identifiers.
+
+For `local` tokens, we encrypt them exclusively using authenticated encryption
+with additional data (AEAD) modes that are also nonce-misuse resistant (NMR).
+This means even if implementations fail to use a secure random number generator
+for the large nonces, message confidentiality isn't imperiled.
+
+### 2. Usability
+
+Developers who are already familiar with JSON Web Tokens (JWT) should be able
+to, intuitively, use PAST in their software with minimal friction.
+
+Additionally, developers who are not already familiar with JWT should be able
+to pick up PAST and use it successfully without introducing security flaws
+into their application.
+
+### 3. Flexibility
+
+Although our library aims to maximize JWT compatibility, out cryptography
+protocols operate over binary strings rather than JSON objects, thereby
+allowing PAST to be used with other serialization standards.
+
+To define these other wire formats, the name of the encoding should be
+suffixed:
+
+* **PAST-JSON** is the default format for payload serialization
+* **PAST-Protobuf** means PAST cryptography using Protobuf serialization
+* **PAST-Raw** uses the cryptography features directly over raw strings
+* **PAST-XML** means PAST cryptography using XML serialization
```

Comment:
I think it may be better to drop this section and stick with a single packed data format (JSON) if PAST is meant to be a crypto hardened replacement for JWT, it would make sense to avoid scope creep and stick as closely as possible to the capabilities of JWT

Reply:
OK.